### DISCUSSION TITLE
Unable to get an organization info from API

### DISCUSSION THREAD

**Discussion by fshuman-a** [planetscale/discussion#632] [Bug Reports] [answered]
Hello,

I'm trying to get detail org info from this API: https://api.planetscale.com/v1/organizations/<orgName> but I keep getting an error 
```
{
    "code": "forbidden",
    "message": "User does not have permission to perform this action."
}
```
I have these permissions selected for the service token: read_databases and read_organization.

If I make an API request to https://api.planetscale.com/v1/organizations, I'm able to get the list of organizations. But I would like a more detailed view.

Is the API working or am I missing a permission (I already tried selecting all read-only scopes but to no avail)?

Thanks!

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @fshuman-a,

This was a bug, we just fixed it. Thank you for reporting!

The payload for that endpoint is the same as returned by the index endpoint. I saw your other discussion looking for billing data, which we do have a solution for, will share more information there soon.

**Reply by fshuman-a** [reply]
Awesome, thanks. I can confirm it works.
